### PR TITLE
Update deposit addresses

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

**Comment by tranvictor** [comment]
We need to parallelize the deposit update process even though it happens only at the beginning of a run. Imagine we have 20 or even 50 tokens.

**Comment by halink0803** [comment]
@tranvictor I've added concurrent when update deposit addresses. Please review.

File: exchange/binance.go [hunk 1]
```diff
@@ -38,7 +38,8 @@ func (self *Binance) UpdateAllDepositAddresses(address string) {
 	}
 }
 
-func (self *Binance) UpdateDepositAddress(token common.Token, address string) {
+func (self *Binance) UpdateDepositAddress(token common.Token, address string, wait *sync.WaitGroup) {
```
Comment (tranvictor) [inline exchange/binance.go]: We should not add wait group param into this function. It is not relevant to Binance scope. It is also the same to other exchanges.

File: cmd/exchanges.go [hunk 2]
```diff
@@ -128,16 +153,22 @@ func NewMainnetExchangePool(addressConfig common.AddressConfig, signer *signer.F
 		switch exparam {
 		case "bittrex":
 			bit := exchange.NewBittrex(bittrex.NewRealBittrexEndpoint(signer), bittrexStorage)
+			wait := sync.WaitGroup{}
 			for tokenID, addr := range addressConfig.Exchanges["bittrex"] {
-				bit.UpdateDepositAddress(common.MustGetToken(tokenID), addr)
+				wait.Add(1)
+				go bit.UpdateDepositAddress(common.MustGetToken(tokenID), addr, &wait)
```
Comment (tranvictor) [inline cmd/exchanges.go]: Shouldn't pass `&wait` to this function, better have another function and signal `Done` inside of that function. Like this:
```
func AsyncUpdateDepositAddress(..., wait) {
   bit.UpdateDepositAddress(common.MustGetToken(tokenID), addr)
   wait.Done()
}
```
and run it in a go routine:
```
go AsyncUpdateDepositAddress(common.MustGetToken(tokenID), addr, &wait)
```

**Pull request merged**